WooCommerce/Classes/ServiceLocator/ProductImageUploader.swift

```diff
@@ -167,6 +173,18 @@ final class ProductImageUploader: ProductImageUploaderProtocol {
         statusUpdatesExcludedProductKeys.remove(key)
     }
 
+    func sendBackgroundUploadNoticeIfNeeded(key: ProductImageUploaderKey, using noticePresenter: NoticePresenter) {
+        if activeUploadsPublisher.contains(key) {
+            let title = NSLocalizedString(
+                "productImageUploader.backgroundUploadNotice.title",
+                value: "Image uploading will continue in the background",
+                comment: ""
+            )
```

Comment:
nit: Consider moving this into `Localization` enum inside an extension.

Reply:
Done in 4b52f71e774660c307536381ee2c6534f19f9439.